cli/azd/pkg/project/framework_service_dotnet.go

```diff
@@ -53,18 +53,8 @@ func (dp *dotnetProject) RequiredExternalTools(context.Context) []tools.External
 	return []tools.ExternalTool{dp.dotnetCli}
 }
 
-// Initializes the docker project
+// Initializes the dotnet project
 func (dp *dotnetProject) Initialize(ctx context.Context, serviceConfig *ServiceConfig) error {
-	if err := dp.dotnetCli.InitializeSecret(ctx, serviceConfig.Path()); err != nil {
-		return err
-	}
-	handler := func(ctx context.Context, args ServiceLifecycleEventArgs) error {
-		return dp.setUserSecretsFromOutputs(ctx, serviceConfig, args)
-	}
-	if err := serviceConfig.AddHandler(ServiceEventEnvUpdated, handler); err != nil {
-		return err
-	}
```

Comment:
If this is a change we really want we should also just deprecate the `IniitizeSecret` and `SetSecret` funcs on the `dotnet` cli wrapper.

Reply:
And remove the `setUserSecretsFromOutputs(...)` func from this file.

Reply:
Correct. Trying to do this minimally for clean diff, if we agree on the direction, I'll log an issue before completing this change.